run-telepresence.sh

```diff
@@ -30,15 +30,15 @@ fi
 
 : ${NAMESPACE:-renku}
 : ${JUPYTERHUB_API_TOKEN:-notebookstoken}
-: ${JUPYTERHUB_BASE_URL:-/jupyterhub/}
-: ${JUPYTERHUB_URL:-http://${MINIKUBE_IP}{JUPYTERHUB_BASE_URL}}
+: ${JUPYTERHUB_SERVICE_PREFIX:-/jupyterhub}
+: ${JUPYTERHUB_URL:-http://${MINIKUBE_IP}{JUPYTERHUB_SERVICE_PREFIX}}
```

Comment:
I actually can't find any place in this service that `JUPYTERHUB_URL` is used

Reply:
ah, I was wondering why it event works with `${MINIKUBE_IP}` there...

Reply:
we certainly can - but JH will still look for it there I think. I'm rewriting this whole service right now, would be a good time to try.

Reply:
You have all my support! 👍